yt_dlp/extractor/bnrnews.py

```diff
@@ -0,0 +1,58 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    str_or_none,
+    url_or_none,
+    urlhandle_detect_ext,
+)
+from ..utils.traversal import traverse_obj
+
+
+class BNRNewsIE(InfoExtractor):
+    _IE_NAME = 'bnrnews'
+    _VALID_URL = r'https?://(?:www\.)?bnrnews\.bg/\w+/post/(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://bnrnews.bg/de/post/394279/bulgarien-heute-2-dezember-2025',
+        'md5': 'eb5a9d9b47347e703c37270124a14169',
+        'info_dict': {
+            'id': '394279',
+            'title': 'Bulgarien heute – 2. Dezember 2025',
+            'thumbnail': 'https://bnrnews.bg/api/media/5168339e-0b81-4d38-9a00-039ef0860035',
+            'url': 'https://bnrnews.bg/api/media/9ce1fff8-f2aa-4d87-90b5-01dff966bb9a',
+            'ext': 'mp3',
+        },
+    }]
+
+    def _make_media_api(self, url_path: str) -> str:
+        return f'https://bnrnews.bg/api/media/{url_path}'
+
+    def _get_single_audio(self, url, video_id):
+        html_page = self._download_webpage(url, video_id, note='Downloading WebPage', errnote='Failed to download page')
+        if html_page:
+            json_body = self._search_nextjs_data(html_page, video_id)
+            if json_body:
+                data_field = traverse_obj(json_body, ('props', 'pageProps', 'data'))
+                if data_field:
```

Comment:
If statements should not be nested like so. We also don't need to guard against cases where nothing useful can be done.